app/helpers/application_helper/button/vm_instance_scan.rb

```diff
@@ -1,6 +1,17 @@
 class ApplicationHelper::Button::VmInstanceScan < ApplicationHelper::Button::Basic
   needs_record
 
+  def calculate_properties
+    super
+    if disabled?
```

Comment:
@chessbyte right, that's my mistake from previous PR... Should have been `disabled?` on both of them. Thanks for noticing the typo! :blush: